### DISCUSSION TITLE
Has anyone compared the two A.I. solutions - Deepstack or CodeProject.AI-Server?

### DISCUSSION THREAD

**Discussion by mkammes** [djdd87/SynoAI#172] [General]
I've been using SynoAI and Deepstack running Docker on my Synology unit for a year now. Half a dozen Reolink 4K cameras. I've been less than impressed with Deepstack - animals and trees/shrubbery/rocks are often misidentified as animals, even with extensive confidence rating changes and boundaries set. People and automobiles, no problem.

Has anyone integrated CodeProject.AI and found its identification to be better...or is it a case of _"it doesn't work better, it just works differently"?_

Thanks!

**Comment by jochenjonc** [comment]
I made the switch to CodeProject.AI and didn't spot any differences (yet). But to be able to do a good comparison both AI engines should be run next to each other (for instance by setting-up a second SynoAI) and then compare the results.
